src/lib/viewers/ActionSummaryViewer.ts

```diff
@@ -0,0 +1,49 @@
+import {Display} from '../Display';
+import {toStyledHeader, indent} from '../utils/StylingUtil';
+import {BundleName, getMessage} from '../messages';
+
+abstract class AbstractActionSummaryViewer {
```

Comment:
There's only one child class right now, but the parent class will be helpful for porting the Run Summary Viewer to the new style.